src/main/webapp/scripts/civic.js

```diff
@@ -7,15 +7,15 @@
  */
 function officialsByLevel(civicObj) {
   let levelsArray = new Array();
-  const levels = {"international" : "International",
-                  "country" : "Country",
-                  "regional" : "Regional",
-                  "administrativeArea2" : "Administrative Area 2",
-                  "administrativeArea1" : "Administrative Area 1",
-                  "locality" : "Locality",
-                  "subLocality2" : "Sublocality 2",
+  const levels = {"special" : "Special",
                   "subLocality1" : "Sublocality 1",
-                  "special" : "Special",};
+                  "subLocality2" : "Sublocality 2",
+                  "locality" : "Locality",
+                  "administrativeArea2" : "County",
+                  "administrativeArea1" : "State",
+                  "regional" : "Regional",
+                   "country" : "Country",
+                  "international" : "International"};
```

Comment:
Didn't these changes already get made in another PR?  Were there some merge issues?

Reply:
Yea, I merged master into this branch, but I don't think master was merged into the base branch (modal-window-integration), so that's why this change is popping up.